RecoLocalCalo/HcalRecProducers/python/HcalHitReconstructor_hbhe_cfi.py

```diff
@@ -31,7 +31,9 @@
     setTimingShapedCutsFlags  = cms.bool(True),
     setTimingTrustFlags       = cms.bool(False), # timing flags currently only implemented for HF
     setPulseShapeFlags        = cms.bool(True),
-    setNegativeFlags          = cms.bool(True), # only in HBHE
```

Comment:
The result would be exactly equivalent. There is a preceding comment explaining why the line was commented out. I don't see how one can get more explicit than that.